tests/integration/tier0/user-prompt-response/test_user_prompt_response.py

```diff
@@ -7,63 +7,41 @@
 def test_check_user_response_user_and_password(convert2rhel):
     """
     Run c2r registration with no username and password provided.
-    Verify that user has to pass non empty username/password string to continue, otherwise enforce the input prompt again.
+    Verify that user has to pass non-empty username/password string to continue, otherwise enforce the input prompt again.
     """
     with convert2rhel("-y --no-rpm-va --serverurl {}".format(env.str("RHSM_SERVER_URL")), unregister=True) as c2r:
-        c2r.expect_exact(" ... activation key not found, username and password required")
-        c2r.expect_exact("Username")
+        c2r.expect(" ... activation key not found, username and password required")
+        c2r.expect("Username")
```

Comment:
Missing timeouts here as well. I would personally add them into this files as well, but leave it up to you. Just please stay consistent (either use them everywhere or nowhere (in the scope of this file).